react/features/filmstrip/functions.native.js

```diff
@@ -90,15 +115,101 @@ export function getColumnCount(stateful: Object | Function) {
 }
 
 /**
- * Returns true if thumbnail reordering is enabled and false otherwise.
+ * Returns true if the filmstrip has a scroll and false otherwise.
  *
  * @param {Object} state - The redux state.
- * @returns {boolean} - True if thumbnail reordering is enabled and false otherwise.
+ * @returns {boolean} - True if the scroll is displayed and false otherwise.
  */
-export function isReorderingEnabled(state) {
-    const { testing = {} } = state['features/base/config'];
-    const enableThumbnailReordering = testing.enableThumbnailReordering ?? true;
+export function isFilmstripScollVisible(state) {
+    if (shouldDisplayTileView(state)) {
+        return state['features/filmstrip']?.tileViewDimensions?.hasScroll;
+    }
 
-    return enableThumbnailReordering;
+    const { aspectRatio, clientWidth, clientHeight, safeAreaInsets = {} } = state['features/base/responsive-ui'];
+    const isNarrowAspectRatio = aspectRatio === ASPECT_RATIO_NARROW;
+    const disableSelfView = shouldHideSelfView(state);
+    const localParticipant = Boolean(getLocalParticipant(state));
+    const localParticipantVisible = localParticipant && !disableSelfView;
+    const participantCount
+        = getParticipantCountWithFake(state)
+            - (localParticipant && (shouldDisplayLocalThumbnailSeparately() || disableSelfView) ? 1 : 0);
+    const { height: thumbnailHeight, width: thumbnailWidth, margin } = styles.thumbnail;
+    const { height, width } = getFilmstripDimensions({
+        aspectRatio,
+        clientWidth,
+        clientHeight,
+        insets: safeAreaInsets,
+        localParticipantVisible
+    });
+
+    if (isNarrowAspectRatio) {
+        return width < (thumbnailWidth + (2 * margin)) * participantCount;
+    }
+
+    return height < (thumbnailHeight + (2 * margin)) * participantCount;
 }
 
+/**
+ * Calculates the width and height of the filmstrip based on the screen size and aspect ratio.
+ *
+ * @param {Object} options - The screen aspect ratio, width, height and safe are insets.
+ * @returns {Object} - The width and the height.
+ */
+export function getFilmstripDimensions({
+    aspectRatio,
+    clientWidth,
+    clientHeight,
+    insets = {},
+    localParticipantVisible = true
+}) {
+    const { height, width, margin } = styles.thumbnail;
+    const isNarrowAspectRatio = aspectRatio === ASPECT_RATIO_NARROW;
+    const filmstripStyle = isNarrowAspectRatio ? styles.filmstripNarrow : styles.filmstripWide;
+
+    if (aspectRatio === ASPECT_RATIO_NARROW) {
+        return {
+            height,
+            width:
+                (shouldDisplayLocalThumbnailSeparately() && localParticipantVisible
+                    ? clientWidth - width - (margin * 2) : clientWidth)
+                    - (insets.left || 0) - (insets.right || 0) - (filmstripStyle.margin * 2)
```

Comment:
Can we make the defaults 0 to avoid having to do the `|| 0 ` all the time?

Reply:
Done